moninom1/FreeRTOS-Plus-TCP
# Add IPv6 specific header files.

Split FreeRTOS_IP header files to IPv4 and IPv6 specific header files.

FreeRTOS_IPv4.h
FreeRTOS_IPv4_Private.h
FreeRTOS_IPv6.h
FreeRTOS_IPv6_Private.h



Test Steps
-----------
As of now compilation test is being done. It might break some test cases which will be fixed in next PR.
```
cmake -S test/build-combination -B test/build-combination/build/ -DTEST_CONFIGURATION=ENABLE_ALL
make -C test/build-combination/build/
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
